docs/set_delegate.md

```diff
@@ -68,7 +81,7 @@ const setDelegate = (key: string) => {
 };
 ```
 
-# How to withdraw delegate
+## How to withdraw delegate
 
 It is possible to `undelegate` by executing a new `setDelegate` operation and not specifying the `delegate` property.
```

Comment:
code example?

Reply:
if you view the whole file, the code example is there. This is just an amendment to the header type